File: Acceso a Datos/Evaluacion_2/WorkSpace2daEvaluacionSpring/demo-thymeleaf/src/main/java/com/maraloedev/Persona.java
package com.maraloedev;

public class Persona {
	
	private String nombre;
	private String apellidos;
	private int edad;
	
	public Persona(String nombre, String apellidos, int edad) {
		this.nombre = nombre;
		this.apellidos = apellidos;
		this.edad = edad;
	}
	
	public String getNombre() {
		return nombre;
	}
	
	public String setNombre(String nombre) {
		return this.nombre = nombre;
	}
	
	public String getApellidos() {
		return apellidos;
	}
	
	public String setApellidos(String apellidos) {
		return this.apellidos = apellidos;
	}
	
	public int getEdad() {
		return edad;
	}
	
	public int setEdad(int edad) {
        return this.edad = edad;
            }
	
	@Override
	public String toString() {
		return "Persona [nombre=" + nombre + ", apellidos=" + apellidos + ", edad=" + edad + "]";
	}
}

File: PMDM/Evaluacion2da/Ejemplos/JetPack_Compose/Ejemplo02_Variables/app/src/main/java/com/maraloedev/ejemplo02_variables/Persona.kt
package com.maraloedev.ejemplo02_variables

import androidx.compose.runtime.MutableState

class Persona(nombre : String, edad : Int, sexo:String) : MutableState<Persona> {
    var nombre : String = nombre
    var edad : Int = edad
    var sexo:String = sexo
    override var value: Persona
        get() = TODO("Not yet implemented")
        set(value) {}

    override fun component1(): Persona {
        TODO("Not yet implemented")
    }

    override fun component2(): (Persona) -> Unit {
        TODO("Not yet implemented")
    }


    override fun toString(): String {
        return "La persona se llama $nombre, tiene $edad años y el sexo es $sexo "
    }
}
